File: src/jcourse/hw/Workers/Worker.java
package jcourse.hw.Workers;

public class Worker {
     String name;
     String position;
     int salary;

    public Worker(String name, String position, int salary) {
        this.name = name;
        this.position = position;
        this.salary = salary;
    }

    String info(int i) {

        return name + " " + position + " " +  i * salary;
    }


}

File: src/kcourse/HW/OOPMethods/Worker.kt
package kcourse.HW.OOPMethods

import java.time.Year
import java.util.*

class Worker(val name:String,val position: String, val year: Int) {

    fun work() = println("Working...")

    val wEx: Int //свойста создаются через объявление переменных val or var
    get() = Calendar.getInstance().get(Calendar.YEAR) - year

}

